modules/installation-network-user-infra.adoc

```diff
@@ -70,18 +77,40 @@ configure a default DNS search zone to allow the API server to resolve the
 node names. Another supported approach is to always refer to hosts by their
 fully-qualified domain names in both the node objects and all DNS requests.
 
-You must configure the network connectivity between machines to allow cluster
+ifndef::ibm-z[]
+[id="installation-host-names-dhcp-user-infra_{context}"]
+== Setting the cluster node host names through DHCP
+
+On {op-system-first} machines, the host name is set through NetworkManager. By default, the machines obtain their host name through DHCP. If the host name is not provided by DHCP, then it is obtained through a reverse DNS lookup. Reverse DNS lookup occurs after the network has been initialized on a node and can take time to resolve. Other system services can start prior to this and detect the host name as `localhost` or similar. You can avoid this by using DHCP to provide the host name for each cluster node.
+
+Additionally, setting the host names through DHCP can bypass any manual DNS record name configuration errors in environments that have a DNS split-horizon implementation.
+endif::ibm-z[]
+
+[id="installation-network-connectivity-user-infra_{context}"]
+== Network connectivity requirements
+
+You must configure the network connectivity between machines to allow {product-title} cluster
 components to communicate. Each machine must be able to resolve the host names
 of all other machines in the cluster.
 
+This section provides details about the ports that are required.
+
+ifndef::restricted,origin[]
+[IMPORTANT]
+====
+In connected {product-title} environments, all nodes are required to have internet access to pull images
+for platform containers and provide telemetry data to Red Hat.
+====
+endif::restricted,origin[]
+
 ifdef::ibm-z-kvm[]
 [NOTE]
 ====
 The {op-system-base} KVM host must be configured to use bridged networking in libvirt or MacVTap to connect the network to the virtual machines. The virtual machines must have access to the network, which is attached to the {op-system-base} KVM host. Virtual Networks, for example network address translation (NAT), within KVM are not a supported configuration.
 ====
 endif::ibm-z-kvm[]
 
-.All machines to all machines
+.Ports used for all machine to all machine communications
```

Comment:
I think "all-machine to all-machine" works better because it helps illustrate the topology. I.e. that all machines communicate to all other machines through these ports.